REQUIRE

```diff
@@ -3,3 +3,4 @@ DataStructures
 Distributions
 DataFrames
 Documenter
+Plots
```

Comment:
Can you add a minimum version number? Just use the currently released version. We should actually do that for all our dependencies, but lets start with any new ones.